time="2019-10-24T23:49:48-07:00" level=debug msg="connect to database "
time="2019-10-24T23:49:48-07:00" level=debug msg="dataSourceName:dbuser:dbuser@tcp(192.168.50.4:30306)/service_config?charset=utf8"
time="2019-10-24T23:49:48-07:00" level=debug msg="FindAll()"
time="2019-10-24T23:50:09-07:00" level=debug msg="user registere failed:dial tcp 192.168.50.4:30306: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond."
